contrib/python-pybind11/template.py

```diff
@@ -0,0 +1,48 @@
+pkgname = "python-pybind11"
+pkgver = "2.11.1"
+pkgrel = 0
+build_style = "cmake"
+hostmakedepends = [
+    "cmake",
+    "ninja",
+    "pkgconf",
+    "python-devel",
+    "python-setuptools",
+]
+makedepends = ["boost-devel"]
+checkdepends = ["python-pytest"]
+pkgdesc = "Seamless operability between C++11 and Python"
+maintainer = "Duncan Bellamy <dunk@denkimushi.com>"
+license = "BSD-3-Clause"
+url = "https://pybind11.readthedocs.io/en/stable/index.html"
+source = f"https://github.com/pybind/pybind11/archive/refs/tags/v{pkgver}.zip"
+sha256 = "b011a730c8845bfc265f0f81ee4e5e9e1d354df390836d2a25880e123d021f89"
+
+
+def post_build(self):
+    from cbuild.build_style import python_module
```

Comment:
we don't import build-styles like this, also python_module is getting removed with 3.12 (all packages are being ported to pep517)

if you wish to execute setup.py, just do it directly

Reply:
okay will change

Reply:
made it a pep517 build and install symlinks for `/usr/share/cmake/pybind11` `/usr/share/pkgconfig/pybind11.pc` `/usr/include/pybind11` etc but dependency checker only finds the package for `/usr/include/pybind11`

Reply:
it's not meant to install those into sitepackages in the first place, but i'm not sure there is any way to fix that (since pep517 doesn't allow it)

i forget if there was a workaround or if the solution is to specialcase these and make them manual `self.do("python" "setup.py" ...` calls

Reply:
All the other distro's I looked at have things installed installed in both locations, arch linux uses symlinks https://gitlab.archlinux.org/archlinux/packaging/packages/pybind11/-/blob/main/PKGBUILD?ref_type=heads

Reply:
yes, but that's a workaround because they are probably never used from there (nothing to my knowledge uses sitepackages/pybind11/{cmake,pkgconfig}). they're just linking it to the actual location- it could also be `mv` and work the same in the end. so if that's the case, might as well actually move it

Reply:
okay will try moving

Reply:
wouldn't this return the include location as inside `site-packages`
https://github.com/pybind/pybind11/blob/eeac2f45728633d7ee6fe792bea2890345697119/pybind11/commands.py#L6 so would need patching

Reply:
hm yes, in that case it's used from pybind itself, makes sense

link is fine then :)